rules/community/gcp/gcp_unauthorized_gke_pod_token_endpoint_usage.yaral

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+rule gcp_unauthorized_gke_pod_token_endpoint_usage {
+
+  meta:
+    author = "Drew Pilarski - Tempus AI"
+    description = "Alerts when an authorization token originating from GKE activity is subsequently used within a command line on an endpoint system, suggesting potential credential exfiltration and reuse."
+    rule_id = "mr_c9a0cb63-0e35-45eb-8b43-ae48385985dc"
+    rule_name = "GCP_Uunauthorized_GKE_Pod_Token_Endpoint_Usage"
+    tactic = "TA0008"
+    technique = "T1550.001"
+    type = "Alert"
+    platform = "GCP"
+    data_source = "Cloud Audit Logs"
+    severity = "Medium"  // Adjust based on your risk assessment
+    priority = "Medium"  // Adjust based on your incident response process
+
+  events:

```

Comment:
Hey @barnabys-drew. Nice rule!

Is there any logic that you can add at the beginning of the events section to filter down the number of events we're working with early on when the rule runs? Maybe `metadata.product_name` or `metadata.log_type`? If not, no worries.